model_lab_configs/huggingface/deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B/1/genai_config.json

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
do we need to check in this? I think this should be generated via modelBuilder pass

Reply:
Auto-generate genai_config has bug... This add some 'reset_session_idx' to workaround.

I think it's hard to tell users where to add clearly in the readme. So I just tell them to replace...